osmaxx-py/excerptconverter/converter_helper.py

```diff
@@ -0,0 +1,49 @@
+from django.contrib import messages
+from django.core.mail import send_mail
+from django.utils.translation import ugettext_lazy as _
+
+import stored_messages
+
+from osmaxx.excerptexport import models
+
+from excerptconverter.baseexcerptconverter import BaseExcerptConverter
+
+class ConverterHelper:
```

Comment:
Consider to have the content as free functions inside this module instead of as instance methods of a class.